src/node.cc

```diff
@@ -3224,6 +3224,21 @@ void SetupProcessObject(Environment* env,
                     Integer::New(env->isolate(),
                                  v8::kPromiseHandlerAddedAfterReject));
 
+  // process._debugOptions
+  Local<Object> debugOptions = Object::New(env->isolate());
+  READONLY_DONT_ENUM_PROPERTY(process,
+                              "_debugOptions",
```

Comment:
https://github.com/nodejs/node/pull/12949#discussion_r118945943
It's not trivial, so I deferred that move. I have no strong feeling about this, maybe someone else does.